_clients/data-prepper/data-prepper-reference.md

```diff
@@ -92,6 +92,42 @@ awsRegion | Conditionally | String | Represents the AWS Region to use ACM or S3.
 authentication | No | Object | An authentication configuration. By default, an unauthenticated server is created for the pipeline. This uses pluggable authentication for HTTPS. To use basic authentication, define the `http_basic` plugin with a `username` and `password`. To provide customer authentication, use or create a plugin that implements [GrpcAuthenticationProvider](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/armeria-common/src/main/java/com/amazon/dataprepper/armeria/authentication/GrpcAuthenticationProvider.java).
 
 
+### s3
+
+This is a source plugin that reads events from S3 objects.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+notification_type | Yes | String | Must be `sqs`
+compression | No | String | The compression algorithm to apply; `none`, `gzip`, or `automatic`. Default is `none`.
+codec | Yes | Codec | The codec to apply. Must be either `newline` or `json`.
+sqs | Yes | sqs | The SQS configuration. See [sqs](#s3-source-sqs) for details.
+aws | Yes | aws | The AWS configuration. See [aws](#s3-source-aws) for details.
```

Comment:
Can we define the acronyms for these?

Reply:
Yes, I can. Do you recommend making any links to them as well? e.g. Link the word "Simple Queue Service" to the SQS product documentation?

Reply:
Yeah, that would be helpful too. We can link to here: https://aws.amazon.com/sqs/